modules/extensions/galasa-extensions-parent/dev.galasa.ras.couchdb/src/main/java/dev/galasa/ras/couchdb/internal/CouchdbDirectoryService.java

```diff
@@ -502,7 +536,12 @@ private JsonObject buildGetRunsQuery(IRasSearchCriteria... searchCriterias) thro
                 RasSearchCriteriaBundle sBundle = (RasSearchCriteriaBundle) searchCriteria;
 
                 inArray(and, "bundle", sBundle.getBundles());
-            } else if (searchCriteria instanceof RasSearchCriteriaResult) {
+            }else if (searchCriteria instanceof RasSearchCriteriaGroup) {
+                RasSearchCriteriaGroup sBundle = (RasSearchCriteriaGroup) searchCriteria;
+
+                inArray(and, "group", sBundle.getGroups());
+            }
+             else if (searchCriteria instanceof RasSearchCriteriaResult) {
```

Comment:
A bit of formatting inconsistency here - the first `else if` is missing a space after the `}`, then the second `else if` is on a separate line to the `}`